components/panel/style.scss

```diff
@@ -133,3 +133,14 @@
 .components-panel .circle-picker {
 	padding-bottom: 20px;
 }
+
+body.rtl .gutenberg .dashicon {
```

Comment:
this should probably go in a dashicons scss: https://github.com/WordPress/gutenberg/tree/master/components/dashicon